src/box/flightrec.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+#include "flightrec.h"
+
+#include "say.h"
+#include "trivia/config.h"
+
+#if defined(ENABLE_FLIGHT_RECORDER)
+# error unimplemented
+#endif
+
+void
+flightrec_init(const char *fr_dirname, size_t logs_size,
+	       size_t logs_max_msg_size, int logs_log_level)
+{
+	(void)fr_dirname;
+	(void)logs_size;
+	(void)logs_max_msg_size;
+	(void)logs_log_level;
+	say_error("Flight recorder is not available in this build");
+}
+
+void
+flightrec_destroy(void)
+{
+}
+
+int
+flightrec_check_log(int64_t logs_size, int64_t logs_max_msg_size,
```

Comment:
Why `check_log`? Should be `flightrec_check_cfg`?

Reply:
I planned to add check for each part of flight recorder (say like for audit: check_format(), check_filter()).